wagtail/admin/templates/wagtailadmin/home/workflow_objects_to_moderate.html

```diff
@@ -99,6 +93,12 @@
     <script src="{% versioned_static 'wagtailadmin/js/modal-workflow.js' %}"></script>
     <script src="{% versioned_static 'wagtailadmin/js/workflow-action.js' %}"></script>
     <script>
-        document.addEventListener('DOMContentLoaded', () => ActivateWorkflowActionsForDashboard('{{ csrf_token|escapejs }}'));
+        document.querySelectorAll('[data-controller="w-dropdown"]').forEach((e) => {
```

Comment:
Technically we could just listen for `w-dropdown:shown` on the document, but that means we can't use `{ once: true }` as the event is dispatched per dropdown (it will only work for the first dropdown opened). This means we'd have to do something more clever in `ActivateWorkflowActionsForDashboard` so that it only adds event listeners to the buttons inside the dropdown once. I tried it but it's worse than the approach I took here (dispatching `w-dropdown:shown` from the specific dropdown, and listening for it here, calling `ActivateWorkflowActionsForDashboard` only once per dropdown).